cumulus_library/template_sql/base_templates.py

```diff
@@ -17,7 +17,12 @@ class TableView(enum.Enum):
     VIEW = "VIEW"
 
 
-def get_base_template(filename_stem: str, path: pathlib.Path | None = None, **kwargs: dict) -> str:
+def get_base_template(*args, **kwargs):
+    """Legacy wrapper for get_template"""
+    return get_template(*args, **kwargs)
+
+
+def get_template(filename_stem: str, path: pathlib.Path | None = None, **kwargs: dict) -> str:
```

Comment:
So are we promoting this to public API? If so:
- do we want to expose it in the main `__init__.py` file then?
- that implies promoting all the "base" templates and macros to be public too - do we have thoughts about stability for that chunk of code across Library releases?
- or maybe it doesn't imply that, and we can maybe remove the default (no `path` provided) behavior of looking in the base template folder. (and make `get_base_template` not just a legacy wrapper, but the version used by Library-internal code, which keeps that default)

Reply:
For posterity:

We're keeping this in the current state, which is a semi-private API that tied to major versions for breaking changes, but we are going to move the function interface to the module level and add a note about intention and caveats. We may make this API fully public in the future.